app/Http/Controllers/Auth/SocialiteController.php

```diff
@@ -164,7 +164,13 @@ private function setRolesFromClaim(string $provider, $user): bool
             ! empty($claims)
         ) {
             $roles = [];
-            $attributes = $this->socialite_user->getRaw();
+            $attributes = [];
+            $attribute_list = $this->socialite_user->getRaw();
+            foreach ($attribute_list as $a) {
```

Comment:
Why are we looping over the attributes twice and not just adjusting the code below?

Reply:
I honestly can't think of another way of doing it. If you have a more elegant solution, feel free to suggest it.

Reply:
The more I think of it, the more perplexed I get by your comment.
How is the code looping over the attributes twice?
The "foreach (Arr::wrap($attributes[$scope]" below is looping over 1 attribute which may have multiple values, it isn't looping over all the attributes again.
Could you explain what you're getting at?